.github/workflows/pythonpublish.yml

```diff
@@ -0,0 +1,32 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+        python-version: 3.7
```

Comment:
Phys2bids supports Python 3.6. Could this be an issue?

Reply:
I don't know, actually. @rmarkello do you think we have to package for 3.6, 3.7, or both?

Reply:
This is just installing the Python distro that will be used to bundle the package; it shouldn't limit which versions of Python versions can install the package, so we should be good!